lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -430,12 +430,18 @@
 
 
       <?php // Custom Share Modal // ?>
-      <?php if ($custom_social_share_link) : ?>
+      <?php if ($custom_social_share_link && $register_voters) : ?>
         <div data-modal id="modal-share" role="dialog">
           <div class="modal__block">
-            <h3><?php print t('Share this Campaign') ?></h3><br>
+            <h3><?php print t('Share This Now!') ?></h3><br>
```

Comment:
yeah I was kind of figuring that next time we use it there will be more fields added to deal with all of this text. but that's a really good idea for now, I'll do some shufflin'